### PR TITLE
docs: add pr review process

### PR DESCRIPTION
**Pull request by Sepehr-Sobhani**
[ZH CARD 2159](https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/gh/bcgov/cas-ciip-portal/2159)

### CODE REVIEW

File: docs/version_control/pull-request-review-process.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Create a pull request with your changes. It's a good idea to create a `draft pull request` first, so that you can get feedback on your changes before you're ready ask for a review. It's also recommended to add the card number as a link to the pull request description, so that it's easy to find the card in the project management tool.
+
+## Step 2: Post in the PR Reviews Teams Channel
+
+Make a post in the `PR Reviews` Teams channel to let others know that you're ready for a review and move your card to `Pending Dev Review` column. It's highly recommended to include the following information:
+
+- The card number and link to the card in the project management tool
+- The link to the pull request
+- Tagging the `PR Reviews` team
+- Any additional context or information that would be helpful for reviewers
```
Comment (pbastia) [inline docs/version_control/pull-request-review-process.md:14]: Here we could have an example of "what is a good PR Reviews post"

File: docs/version_control/pull-request-review-process.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Step 7: Move the card to `Pending PO Approval` column
+
+Before doing this, make sure that the code has been deployed to the dev environment by looking at [shipit](https://cas-shipit.apps.silver.devops.gov.bc.ca/bcgov/cas-cif/dev)
+
+## **While waiting for a review / After posting the card for a review**
+
+While you're waiting for a review, you can decide to pull in a new card and start working on that or review other pull requests depending on how many cards you have in progress, how many cards are awaiting review, and how many cards are in the backlog.
+In general, It's a good practice to check if anything needs to be reviewed before pulling a new card into the sprint.
+
```
Comment (pbastia) [inline docs/version_control/pull-request-review-process.md:48]: Pulling a new card into the sprint requires PO approval (taking a card from the sprint backlog doesn't)

File: docs/version_control/pull-request-review-process.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+## **Nice to know**
+
+- Posts can disappear from the channel if there are too many posts, It's a good idea to bump your post to keep it visible.
+- If the pull request has a lot of changes, suggesting a walkthrough of the code to the reviewer can be helpful.
```
Comment (pbastia) [inline docs/version_control/pull-request-review-process.md:52]: ...Or try to limit the amount of changes per PR, split work where possible ;)

File: docs/version_control/pull-request-review-process.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+Make any necessary changes to your code based on the feedback from the reviewers. You can also respond to the comments and questions to provide additional context or clarify any confusion.
+
+## Step 5: Reviewers Approve or Request Changes
+
+Once you have made changes, the reviewers will review your code again. They can either approve your changes or request additional changes. If they request changes, go back to Step 4 and repeat the process until your changes are approved.
+
+## Step 6: Merge the Pull Request
+
+Once your changes have been approved and the code is updated with the latest changes on the `develop` branch, you can merge the pull request.
```
Comment (pbastia) [inline docs/version_control/pull-request-review-process.md:38]: Documentation about rebasing process before merging changes

**Review by pbastia — CHANGES_REQUESTED** [review]
Thanks for taking that on! Looks great, there are just a few items missing

File: docs/version_control/pull-request-review-process.md (hunk 1 above)
Comment (Sepehr-Sobhani) [reply docs/version_control/pull-request-review-process.md:14]: @pbastia Do you mean something like a screenshot from one of our existing PR review posts?!

Comment (pbastia) [reply docs/version_control/pull-request-review-process.md:14]: Yes, or just a text version of that screenshot, in a frame. something like

> `@PR Reviews` this pull request to do XYZ 
> PR: [link to GH PR](https://github.com/bcgov/cas-home/pull/8) 
> Card: ...

etc

**Review by pbastia — APPROVED** [review]

**Pull request merged**